Web Inspector: rename InspectorController to PageInspectorController

burg opened the pull request:
#### 8f1f2227c8a7ea968db72b3de47f3408fe1f573d
<pre>
Web Inspector: rename InspectorController to PageInspectorController
<a href="https://bugs.webkit.org/show_bug.cgi?id=302479">https://bugs.webkit.org/show_bug.cgi?id=302479</a>

Reviewed by Devin Rousso.

This aligns the naming with other InspectorControllers, each of which is
associated with a target type but do not share common base class. Reflect
this by prepending &apos;Page&apos; to the name.

No new tests, just a name change.

* Source/WebCore/Modules/mediacontrols/MediaControlsHost.cpp:
* Source/WebCore/SaferCPPExpectations/ForwardDeclCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncheckedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/bindings/js/JSDOMWindowBase.cpp:
* Source/WebCore/dom/Node.cpp:
* Source/WebCore/inspector/FrameInspectorController.cpp:
* Source/WebCore/inspector/InspectorBackendClient.h:
* Source/WebCore/inspector/InspectorFrontendAPIDispatcher.cpp:
* Source/WebCore/inspector/InspectorFrontendClientLocal.cpp:
(WebCore::InspectorBackendDispatchTask::create):
(WebCore::InspectorBackendDispatchTask::InspectorBackendDispatchTask):
(WebCore::InspectorFrontendClientLocal::InspectorFrontendClientLocal):
(WebCore::InspectorFrontendClientLocal::protectedInspectedPageController const):
(WebCore::InspectorFrontendClientLocal::restoreAttachedWindowHeight):
* Source/WebCore/inspector/InspectorFrontendClientLocal.h:
* Source/WebCore/inspector/InspectorFrontendHost.cpp:
(WebCore::InspectorFrontendHost::isBeingInspected):
* Source/WebCore/inspector/InspectorInstrumentation.cpp:
* Source/WebCore/inspector/InspectorNetworkIntercept.h:
* Source/WebCore/inspector/InspectorOverlay.cpp:
(WebCore::InspectorOverlay::InspectorOverlay):
* Source/WebCore/inspector/InspectorOverlay.h:
* Source/WebCore/inspector/InspectorResourceUtilities.cpp:
* Source/WebCore/inspector/PageDebugger.cpp:
* Source/WebCore/inspector/PageInspectorController.cpp: Renamed from Source/WebCore/inspector/InspectorController.cpp.
* Source/WebCore/inspector/PageInspectorController.h: Renamed from Source/WebCore/inspector/InspectorController.h.
* Source/WebCore/inspector/WorkerInspectorController.cpp:
* Source/WebCore/inspector/agents/InspectorDOMAgent.cpp:
* Source/WebCore/inspector/agents/page/PageTimelineAgent.cpp:
* Source/WebCore/loader/EmptyClients.cpp:
* Source/WebCore/loader/FrameLoader.cpp:
* Source/WebCore/page/ContextMenuController.cpp:
* Source/WebCore/page/FrameConsoleClient.cpp:
* Source/WebCore/page/LocalFrameView.cpp:
* Source/WebCore/page/Page.cpp:
(WebCore::Page::Page):
(WebCore::Page::protectedInspectorController):
* Source/WebCore/page/Page.h:
(WebCore::Page::inspectorController):
* Source/WebCore/page/PageDebuggable.cpp:
* Source/WebCore/testing/Internals.cpp:
* Source/WebKit/WebProcess/Inspector/RemoteWebInspectorUI.cpp:
* Source/WebKit/WebProcess/Inspector/WebFrameInspectorTarget.cpp:
* Source/WebKit/WebProcess/Inspector/WebInspectorBackendClient.cpp:
(WebKit::WebInspectorBackendClient::openLocalFrontend):
* Source/WebKit/WebProcess/Inspector/WebInspectorBackendClient.h:
* Source/WebKit/WebProcess/Inspector/WebInspectorInternal.cpp:
* Source/WebKit/WebProcess/Inspector/WebInspectorUI.cpp:
* Source/WebKit/WebProcess/Inspector/WebInspectorUI.h:
* Source/WebKit/WebProcess/Inspector/WebPageInspectorTarget.cpp:
* Source/WebKit/WebProcess/WebPage/WebPage.cpp:
* Source/WebKitLegacy/ios/WebCoreSupport/WebInspectorClientIOS.mm:
(WebInspectorClient::openLocalFrontend):
(WebInspectorFrontendClient::WebInspectorFrontendClient):
* Source/WebKitLegacy/mac/WebCoreSupport/WebInspectorClient.h:
* Source/WebKitLegacy/mac/WebCoreSupport/WebInspectorClient.mm:
(WebInspectorClient::openLocalFrontend):
(WebInspectorFrontendClient::WebInspectorFrontendClient):
* Source/WebKitLegacy/mac/WebInspector/WebInspector.mm:
* Source/WebKitLegacy/mac/WebInspector/WebNodeHighlight.h:
* Source/WebKitLegacy/mac/WebInspector/WebNodeHighlight.mm:
(-[WebNodeHighlight initWithTargetView:inspectorController:]):
* Source/WebKitLegacy/mac/WebInspector/WebNodeHighlightView.mm:

Canonical link: <a href="https://commits.webkit.org/303228@main">https://commits.webkit.org/303228@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b2a56d6e3ef9a45d08ba526f3a8d9f399c418186

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/131764 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/4256 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/42767 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/139273 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/83636 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | ⏳ 🛠 ios-apple 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/133634 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/4185 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/4018 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/100716 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/83636 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/993ed59f-2d7a-4269-a88f-78619447f6c3) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/134710 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/162/builds/3007 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/118013 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/81499 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/4d13c956-1e7b-46e9-b5dc-859f8e4d01dd) 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/154/builds/2894 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac-debug~~](https://ews-build.webkit.org/#/builders/165/builds/740 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/82495 "Built successfully") | | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/111628 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/36140 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/141919 "Built successfully") | | 
| | [  ~~🛠 vision~~](https://ews-build.webkit.org/#/builders/153/builds/3925 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/36717 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/109087 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/4006 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/161/builds/3453 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/109249 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/27667 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/2983 "Passed tests") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/114294 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/57135 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/3979 "Built successfully") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/32704 "Passed tests") | | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/157/builds/3811 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/67426 "Built successfully") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/163/builds/4071 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/3939 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

the-chenergy commented on Source/WebCore/inspector/InspectorFrontendClientLocal.cpp:
```diff
@@ -80,7 +80,7 @@ static const float minimumAttachedInspectedWidth = 320.0f;
     WTF_MAKE_TZONE_ALLOCATED(InspectorBackendDispatchTask);
 public:
-    static Ref<InspectorBackendDispatchTask> create(InspectorController* inspectedPageController, InspectorFrontendClientLocal::DispatchBackendTarget dispatchTarget)
+    static Ref<InspectorBackendDispatchTask> create(PageInspectorController* inspectedPageController, InspectorFrontendClientLocal::DispatchBackendTarget dispatchTarget)
```
I think you should make a drive-by fix to rename all `inspectedPageController` to `pageInspectorController` as well (or just `inspectorController`, if the scope is short). I don't think the slight reordering in the name adds value.

```suggestion
    static Ref<InspectorBackendDispatchTask> create(PageInspectorController* pageInspectorController, InspectorFrontendClientLocal::DispatchBackendTarget dispatchTarget)
```

burg replied:
I'll need to revisit these files shortly to adopt CheckedRef for InspectorController, so let's keep this a simple rename.

burg committed:
Web Inspector: rename InspectorController to PageInspectorController
https://bugs.webkit.org/show_bug.cgi?id=302479

Reviewed by Devin Rousso.

This aligns the naming with other InspectorControllers, each of which is
associated with a target type but do not share common base class. Reflect
this by prepending 'Page' to the name.

No new tests, just a name change.

* Source/WebCore/Modules/mediacontrols/MediaControlsHost.cpp:
* Source/WebCore/SaferCPPExpectations/ForwardDeclCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncheckedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/bindings/js/JSDOMWindowBase.cpp:
* Source/WebCore/dom/Node.cpp:
* Source/WebCore/inspector/FrameInspectorController.cpp:
* Source/WebCore/inspector/InspectorBackendClient.h:
* Source/WebCore/inspector/InspectorFrontendAPIDispatcher.cpp:
* Source/WebCore/inspector/InspectorFrontendClientLocal.cpp:
(WebCore::InspectorBackendDispatchTask::create):
(WebCore::InspectorBackendDispatchTask::InspectorBackendDispatchTask):
(WebCore::InspectorFrontendClientLocal::InspectorFrontendClientLocal):
(WebCore::InspectorFrontendClientLocal::protectedInspectedPageController const):
(WebCore::InspectorFrontendClientLocal::restoreAttachedWindowHeight):
* Source/WebCore/inspector/InspectorFrontendClientLocal.h:
* Source/WebCore/inspector/InspectorFrontendHost.cpp:
(WebCore::InspectorFrontendHost::isBeingInspected):
* Source/WebCore/inspector/InspectorInstrumentation.cpp:
* Source/WebCore/inspector/InspectorNetworkIntercept.h:
* Source/WebCore/inspector/InspectorOverlay.cpp:
(WebCore::InspectorOverlay::InspectorOverlay):
* Source/WebCore/inspector/InspectorOverlay.h:
* Source/WebCore/inspector/InspectorResourceUtilities.cpp:
* Source/WebCore/inspector/PageDebugger.cpp:
* Source/WebCore/inspector/PageInspectorController.cpp: Renamed from Source/WebCore/inspector/InspectorController.cpp.
* Source/WebCore/inspector/PageInspectorController.h: Renamed from Source/WebCore/inspector/InspectorController.h.
* Source/WebCore/inspector/WorkerInspectorController.cpp:
* Source/WebCore/inspector/agents/InspectorDOMAgent.cpp:
* Source/WebCore/inspector/agents/page/PageTimelineAgent.cpp:
* Source/WebCore/loader/EmptyClients.cpp:
* Source/WebCore/loader/FrameLoader.cpp:
* Source/WebCore/page/ContextMenuController.cpp:
* Source/WebCore/page/FrameConsoleClient.cpp:
* Source/WebCore/page/LocalFrameView.cpp:
* Source/WebCore/page/Page.cpp:
(WebCore::Page::Page):
(WebCore::Page::protectedInspectorController):
* Source/WebCore/page/Page.h:
(WebCore::Page::inspectorController):
* Source/WebCore/page/PageDebuggable.cpp:
* Source/WebCore/testing/Internals.cpp:
* Source/WebKit/WebProcess/Inspector/RemoteWebInspectorUI.cpp:
* Source/WebKit/WebProcess/Inspector/WebFrameInspectorTarget.cpp:
* Source/WebKit/WebProcess/Inspector/WebInspectorBackendClient.cpp:
(WebKit::WebInspectorBackendClient::openLocalFrontend):
* Source/WebKit/WebProcess/Inspector/WebInspectorBackendClient.h:
* Source/WebKit/WebProcess/Inspector/WebInspectorInternal.cpp:
* Source/WebKit/WebProcess/Inspector/WebInspectorUI.cpp:
* Source/WebKit/WebProcess/Inspector/WebInspectorUI.h:
* Source/WebKit/WebProcess/Inspector/WebPageInspectorTarget.cpp:
* Source/WebKit/WebProcess/WebPage/WebPage.cpp:
* Source/WebKitLegacy/ios/WebCoreSupport/WebInspectorClientIOS.mm:
(WebInspectorClient::openLocalFrontend):
(WebInspectorFrontendClient::WebInspectorFrontendClient):
* Source/WebKitLegacy/mac/WebCoreSupport/WebInspectorClient.h:
* Source/WebKitLegacy/mac/WebCoreSupport/WebInspectorClient.mm:
(WebInspectorClient::openLocalFrontend):
(WebInspectorFrontendClient::WebInspectorFrontendClient):
* Source/WebKitLegacy/mac/WebInspector/WebInspector.mm:
* Source/WebKitLegacy/mac/WebInspector/WebNodeHighlight.h:
* Source/WebKitLegacy/mac/WebInspector/WebNodeHighlight.mm:
(-[WebNodeHighlight initWithTargetView:inspectorController:]):
* Source/WebKitLegacy/mac/WebInspector/WebNodeHighlightView.mm:

Canonical link: https://commits.webkit.org/303228@main
